src/utils/getTimeDifference.ts

```diff
@@ -32,11 +32,12 @@ export const getTimeDifference = (createdAt: string): string => {
   }
 
   const diffInWeeks = now.diff(targetTime, 'week')
-  if (diffInWeeks < 4) {
+  const diffInMonths = now.diff(targetTime, 'month')
+
+  if (diffInWeeks < 4 || (diffInWeeks === 4 && diffInMonths < 1)) {
```

Comment:
I think it is better to mention in the comments about the exact reason for doing this.